apps/backend/src/database/migrate.ts

```diff
@@ -0,0 +1,38 @@
+import { MigrationExecutor } from 'typeorm';
+import db from './datasource';
+import { Context } from 'aws-lambda';
+
+/* eslint-disable @typescript-eslint/no-unused-vars */
+export const handler = async (event?: unknown, context?: Context) => {
+  console.log('Starting migrations...');
+  try {
+    if (!db.isInitialized) {
+      await db.initialize();
+    }
+
+    const migrationExecutor = new MigrationExecutor(db, db.createQueryRunner());
+
+    const executed = await migrationExecutor.getExecutedMigrations();
+    const pending = await migrationExecutor.getPendingMigrations();
+
+    console.log('---> executed:');
+    console.log(executed.map((mig) => mig.name));
+
+    console.log('---> pending:');
+    console.log(pending.map((mig) => mig.name));
+
```

Comment:
Do we want to log these to the daily log files?

Reply:
I don't think so.  The logs will be in cloudwatch as lambda runs